massa-api-exports/src/lib.rs

```diff
@@ -80,3 +81,10 @@ pub enum ListType {
     /// contains allowed entry
     Whitelist,
 }
+
+/// Wrap request params into struct for ApiV2 method
+#[derive(Deserialize, Serialize)]
```

Comment:
Why do we wrap the PageRequest here? To have a new struct we can freely make changes to for APIv2 without breaking the APIv1?

Reply:
We should have an evolutive struct in APIV2, this is a good practice in API design, for example, if we would add Sort struct without breaking our existing API, we'll just add to ApiRequest. In addition, JsonRPC takes an array of objects as query parameters:

BAD
```shell
curl --location --request POST 'http://localhost:33036' \
--header 'Content-Type: application/json' \
--data-raw '{
    "jsonrpc": "2.0",
    "id": 1,
    "method": "get_largest_stakers",
    "params": [ {{"sort_by_id": "ASC", "offset": 0, "limit": 2 }]
}'
```
We should play this flatten object and bind it manually to our structs: Sort PageRequest. if we have same parameters names between different args, here we are ...

BETTER

```shell
curl --location --request POST 'http://localhost:33036' \
--header 'Content-Type: application/json' \
--data-raw '{
    "jsonrpc": "2.0",
    "id": 1,
    "method": "get_largest_stakers",
    "params": [ {"sort": {"id": "ASC"}, "page_request": {"offset": 0, "limit": 2 }}]
}'
```
Nothing to do, ApiRequest will hold all for us.

Reply:
Understood, it's what I thought ;). Nice addition then 👍